api/envoy/config/core/v3/base.proto

```diff
@@ -358,8 +358,10 @@ message HeaderValueOption {
   // existing values. Otherwise it replaces any existing values.
   google.protobuf.BoolValue append = 2;
```

Comment:
We should be able to mark this as deprecated.

Reply:
+1

Reply:
of course. 😸